dp/utils/logging.py

```diff
@@ -14,5 +14,13 @@
 
 
 def get_logger(name: str) -> Logger:
+    """
+
+    Args:
+      name: str: 
```

Comment:
Incomplete explanation of parameter in docstring.

Reply:
Updated.